web3/contract.py

```diff
@@ -268,6 +270,12 @@ def deploy(cls, transaction=None, args=None, kwargs=None):
         txn_hash = cls.web3.eth.sendTransaction(deploy_transaction)
         return txn_hash
 
+    def fallback(self):
+        if self._fallback:
+            return self._fallback
+        else:
+            raise FallbackNotFound("Fallback function does not exist.")
```

Comment:
I think a slightly more informative error message would be

> No fallback function was found in the contract ABI

Reply:
Sure, I will change it.